tokio-i3ipc/examples/get_tree.rs

```diff
@@ -2,7 +2,7 @@ use std::io;
 
 use tokio_i3ipc::{reply, I3};
 
-#[tokio::main(basic_scheduler)]
+#[tokio::main]
```

Comment:
Would prefer to use the single threaded scheduler for the examples

Reply:
Makes sense for this example.